src/db.rs

```diff
@@ -70,11 +70,12 @@ impl DB {
         Ok(())
     }
 
-    pub fn migrate<P: AsRef<Path>>(&mut self, path: P) -> Result<()> {
-        if !self.dirs.is_empty() {
+    pub fn migrate<P: AsRef<Path>>(&mut self, path: P, merge: bool) -> Result<()> {
+        if !self.dirs.is_empty() && !merge {
             bail!(
                 "To prevent conflicts, you can only migrate from z with an empty \
-                 zoxide database!"
+                 zoxide database!\n\
```

Comment:
I wish this could look nicer, but we don't want overlong lines, especially in a terminal application.

Reply:
[Rust does support multiline strings](https://www.programming-idioms.org/idiom/48/multi-line-string-literal/435/rust), so there isn't any need for the `\` and `\n`.

Reply:
Indeed it does. But then you get code that looks like:

```rust
            bail!(
                "To prevent conflicts, you can only migrate from z with an empty \
                 zoxide database!
If you wish to merge the two, specify the `--merge` flag."
            );
```

which I'm not sure is much better.

Another option would be to use `format!("{}\n{}", "first", "second")`. Let me know if your opinion is unchanged and you would still prefer the multiline string above.